governance.md

```diff
@@ -0,0 +1,162 @@
+
+## Relation to the Django Software Foundation
+
+Django Commons is an independent initiative from the Django Software Foundation (DSF). We are here for the third-party packages that compliment Django. We are here as a community for and by Django users.
+
+## Teams
+
+There are four teams that work together to handle the operations of Django Commons. 
+
+They are:
+
+- Admin Team
+- CoC Team
+- PyPI Team
+- Security Team
+- Communication
+
+Our teams use two main ways to communicate: GitHub Discussions and team related Google Group email.
+
+Quick guide:
+
+- Use GitHub Discussions for: technical questions, project ideas, general help, and community discussions.
+- Use GitHub private vulnerability reporting for: security concerns.
+- Use Google Group email for: personal matters, sensitive information, or private volunteer concerns.
+
+We believe in being open and transparent, so we encourage everyone to use GitHub Discussions as their first choice. This public space lets everyone see and join our conversations, which helps new volunteers learn from past discussions and feel included in our community.
+
+## Admin team
+
+This is the core team that is responsible for the overall management of the organization. They have admin access to the GitHub organization and are responsible for making sure that Django Commons is running smoothly.
+
+Responsibilities/tasks include:
+
+- Manage incoming transfers and outgoing transfers of projects.
+- Manage new membership applications.
+- Respond to inquiries from the community on the [Discussion forum](https://github.com/orgs/django-commons/discussions).
+- Manage all team rotation schedules, making sure that every team is actively staffed in a sustainable manner.
+- Encourage people from diverse backgrounds to join one of the teams.
+- Provide resources and facilitate projects seeking new or additional maintainers. Help find new maintainers for projects that need them.
+- Facilitate discovery of maintenance best practices.
+- Maintain the [django-commons/membership](https://github.com/django-commons/membership) repository.
+- Maintain the [django-commons/controls](The controls for managing Django Commons projects) repository.
+- Conduct check-ins with project maintainers and our community. What else can we do to help our maintainers?
+- Maintain this document and other documents that describe how Django Commons operates.
+- Attend regular team meetings to discuss the state of Django Commons and what we can do to improve it.
+- Help organize events and other community activities.
+- Reach out to projects that might be interested in joining Django Commons.
+
+Out of scope responsibilities/tasks:
+
+- Maintaining projects. This is the responsibility of the maintainers of the projects themselves. However, admin team members are free to contribute to projects as they see fit. There is just no expectation that they do so.
+
+Qualifications for admin members:
+
+- Be a DSF member.
```

Comment:
```suggestion
- Be a [DSF individual member](https://www.djangoproject.com/foundation/individual-members/).
```
This may be helpful since not everyone know what a DSF individual member is.